dnfdragora/const.py

```diff
@@ -134,14 +134,15 @@
 }
 
 RPM_ACTIONS = {
+    '10': _("Running scriptlet: %s"),
```

Comment:
This is coming from dnfdaemon, is it not? If so, we should make dnfdaemon enumerate `10` with a real value.